apps/emqx_connector/src/schema/emqx_connector_ee_schema.erl

```diff
@@ -108,6 +114,30 @@ connector_structs() ->
                     desc => <<"Syskeeper Proxy Connector Config">>,
                     required => false
                 }
+            )},
+        {pgsql,
+            mk(
+                hoconsc:map(name, ref(emqx_bridge_pgsql, "config_connector")),
+                #{
+                    desc => <<"PostgreSQL Connector Config">>,
+                    required => false
+                }
+            )},
+        {timescale,
+            mk(
+                hoconsc:map(name, ref(emqx_bridge_timescale, "config_connector")),
+                #{
+                    desc => <<"Timescale Connector Config">>,
+                    required => false
+                }
+            )},
+        {matrix,
```

Comment:
Nit: keep sorted

Reply:
I think this will go away when we have implemented something similar to emqx_action_info for connectors